src/components/core/MeshUIComponent.js

```diff
@@ -377,6 +381,10 @@ export default function MeshUIComponent( Base = class {} ) {
             // attribute from the light batch.
 
             for ( const prop of Object.keys(options) ) {
+                if(this[prop] == options[prop])
```

Comment:
@felixmariotto this change is unrelated to bestFit, so feel free to revert it if this wasn't done by design. 

I thought it was redundant to force a component update if the value being updated is the same as the existing one, since the outcome wouldn't change. This should improve performance in certain situations.